_notices/rsn0047.md

```diff
@@ -42,6 +42,24 @@ the new strategy will be implemented across all RAPIDS libraries.
 
 ## Impact
 
+After a repository has switched to the new strategy, existing local clones of
+repository may need to be updated to use the new `main` branch.

```

Comment:
Is there a nice `gh` command to change the default branch on our forks, too? I see this prompt on my fork:

![{E46C4B76-0048-4A85-8FE4-7267A5E0986B}](https://github.com/user-attachments/assets/8312dc66-dedc-413c-b41d-7cf4ef47df3d)


Maybe: https://stackoverflow.com/questions/74960450/change-default-branch-via-cli

Reply:
I added a section below with instructions for updating your fork